### PR TITLE
fixed spelling

### PR DESCRIPTION
**Pull request by kaikreuzer**
Signed-off-by: Kai Kreuzer <kai@openhab.org>

### CODE REVIEW

**Comment by wborn** [comment]
Can you also change:

* "azure" to "Azure"
* "IOT"/"Iot" to "IoT"

in the pom.xml and README.md of the addon?

```
addons/io/org.openhab.io.azureiothub/pom.xml:  <name>azure IOT Hub Connector</name>
addons/io/org.openhab.io.azureiothub/README.md:# azure IOT Hub Connector
addons/io/org.openhab.io.azureiothub/README.md:The azure IOT Hub connector replicates your local things to a [microsoft azure IOT Hub]
addons/io/org.openhab.io.azureiothub/README.md:(https://azure.microsoft.com/en-us/services/iot-hub/).
addons/io/org.openhab.io.azureiothub/README.md:This IOT building block resides in the azure cloud and allows 2 way communication with your devices and your IOT infrastructure in the cloud.
addons/io/org.openhab.io.azureiothub/README.md:The IOT hub can be connected to other azure services like a [database] (https://azure.microsoft.com/en-us/services/hdinsight/), [stream analytics] (https://azure.microsoft.com/en-us/services/stream-analytics/), [machine learning] (https://azure.microsoft.com/en-us/services/machine-learning/), [time series insights] (https://azure.microsoft.com/en-us/services/time-series-insights/)
addons/io/org.openhab.io.azureiothub/README.md:If you want to log more than this, you can switch to a paying price tier: [pricing details] (https://azure.microsoft.com/en-us/pricing/details/iot-hub/)
addons/io/org.openhab.io.azureiothub/README.md:[Create your Azure account] (https://azure.microsoft.com/en-us/free/)
addons/io/org.openhab.io.azureiothub/README.md:From the [azure portal] (http://portal.azure.com/), you need to create an Iot Hub. Click on the '+' sign on the upper left, search for 'IOT Hub', click create and follow the wizard. Once the hub is available, go to settings - Shared access policies, click `iothubowner` and copy the connection string - primary key.
addons/io/org.openhab.io.azureiothub/README.md:The openHAB Azure IoT Hub Connector can operate in 2 modes:
addons/io/org.openhab.io.azureiothub/README.md:In publish mode, openHAB will sync all its devices and its status changes to Azure.

**Comment by kaikreuzer** [comment]
Good point. All fixed now. Also fixed the broken links in the README.

File: addons/io/org.openhab.io.azureiothub/README.md [hunk 1]
```diff
@@ -1,22 +1,23 @@
-# azure IOT Hub Connector
+# Azure IoT Hub Connector
 
-The azure IOT Hub connector replicates your local things to a [microsoft azure IOT Hub]
-(https://azure.microsoft.com/en-us/services/iot-hub/).
-This IOT building block resides in the azure cloud and allows 2 way communication with your devices and your IOT infrastructure in the cloud.
-The IOT hub can be connected to other azure services like a [database] (https://azure.microsoft.com/en-us/services/hdinsight/), [stream analytics] (https://azure.microsoft.com/en-us/services/stream-analytics/), [machine learning] (https://azure.microsoft.com/en-us/services/machine-learning/), [time series insights] (https://azure.microsoft.com/en-us/services/time-series-insights/)
+The Azure IoT Hub connector replicates your local things to a [Microsoft Azure IoT Hub](https://azure.microsoft.com/en-us/services/iot-hub/).
+This IoT building block resides in the azure cloud and allows 2 way communication with your devices and your IoT infrastructure in the cloud.
```
Comment (wborn) [inline addons/io/org.openhab.io.azureiothub/README.md]: "azure" -> "Azure"

File: addons/io/org.openhab.io.azureiothub/README.md [hunk 2]
```diff
@@ -1,22 +1,23 @@
...
+The IoT hub can be connected to other Azure services like a [database](https://azure.microsoft.com/en-us/services/hdinsight/), [stream analytics](https://azure.microsoft.com/en-us/services/stream-analytics/), [machine learning](https://azure.microsoft.com/en-us/services/machine-learning/) and [time series insights](https://azure.microsoft.com/en-us/services/time-series-insights/).
 
 ## Pricing
 
-You may send up to 8000 messages a day for free. that is 5.5 status updates every minute. Anything more then this in the free price tier is simply neglected by the IOT hub.
-If you want to log more than this, you can switch to a paying price tier: [pricing details] (https://azure.microsoft.com/en-us/pricing/details/iot-hub/)
+You may send up to 8000 messages a day for free. That is 5.5 status updates every minute. Anything more then this in the free price tier is simply neglected by the IoT Hub.
```
Comment (wborn) [inline addons/io/org.openhab.io.azureiothub/README.md]: "more then this" -> "more than this"
Can you also add a new line at the end of each sentence?

**Comment by kaikreuzer** [comment]
Sure, updated!

**Pull request merged**